README.md

```diff
@@ -6,23 +6,37 @@
 [![Coverage Status](https://coveralls.io/repos/github/nhsuk/profiles/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/profiles?branch=master)
 [![Known Vulnerabilities](https://snyk.io/test/github/nhsuk/profiles/badge.svg)](https://snyk.io/test/github/nhsuk/profiles)
 
-A collection of health service provider profiles. Initially containing GP
+> A collection of health service provider profiles. Initially containing GP
 practices and Pharmacies.
 
+## Test environments
+
+As the application is being developed, every Pull Request has its own test
+environment automatically built and deployed to. The path to the environment
+is added as a comment to the PR in GitHub.
```

Comment:
This is not true - the link is in the checks (as the Details for rancher/deployment check).

Reply:
Add as an issue, nout to do with this change.

Reply:
My bad. Sorted this and alphabetised the docker config to make it easier to search through.